README.md

```diff
@@ -171,10 +178,10 @@ Usage of graphite-ch-optimizer:
   -c, --config string                Filename of the custom config. CLI arguments override it
       --print-defaults               Print default config values and exit
   -v, --version                      Print version and exit
-      --optimize-interval duration   The active partitions won't be optimized more than once per this interval, seconds (default 72h0m0s)
+      --optimize-interval duration   The partition will be merged unless it have active writes for more than, duration (default 72h0m0s)
   -s, --server-dsn string            DSN to connect to ClickHouse server (default "tcp://localhost:9000?&optimize_throw_if_noop=1&receive_timeout=3600&debug=true")
   -n, --dry-run                      Will print how many partitions would be merged without actions
-      --loop-interval duration       Daemon will check if there partitions to merge once per this interval, seconds (default 1h0m0s)
+      --loop-interval duration       Daemon will check if there partitions to merge once per this interval, duration (default 1h0m0s)
```

Comment:
`The daemon will check if there partitions to merge once per this interval (default 1h0m0s)`

Adding the duration word here makes it confusing, imo

Reply:
You are right. It's redundant since the type is already defined in `--loop-interval duration`. Thanks